Saas/Set-BcNotificationRecipient.ps1

```diff
@@ -0,0 +1,54 @@
+<#
+ .Synopsis
+  Function for creatig a notification recipient for a Business Central tenant.
+ .Description
+  Function for creating a notification recipient for a Business Central tenant.
+  Submit Email and Name.
+  Wrapper for https://learn.microsoft.com/en-us/dynamics365/business-central/dev-itpro/administration/administration-center-api_notifications#create-notification-recipient
+ .Parameter bcAuthContext
+  Authorization Context created by New-BcAuthContext.
+ .Parameter apiVersion
+  API version. Default is v2.15.
+ .Example
+  $bcauthContext = New-BcAuthContext -includeDeviceLogin
+  Get-BcNotificationRecipients -bcAuthContext $bcauthContext
+#>
+
+function Set-BcNotificationRecipient {
```

Comment:
is this setting all recipients - or adding one recipient?
If it is setting all - then please use Set-BcNotificationRecipients
If it is adding one - then please use New-BcNotificationRecipient